AK Zeus/app/src/main/java/ayds/zeus/songinfo/home/view/SongToReleaseDateStringMapper.kt

```diff
@@ -0,0 +1,41 @@
+package ayds.zeus.songinfo.home.view
+import ayds.zeus.songinfo.home.model.entities.Song
+
+
+interface SongToReleaseDateStringMapper {
+    fun map(song: Song): String
+}
+
+internal class SongToDayReleaseDateStringMapper: SongToReleaseDateStringMapper {
+    override fun map(song: Song): String {
+        val dateComponents = song.releaseDate.split("-")
+        return "${dateComponents[2]}/${dateComponents[1]}/${dateComponents[0]}"
```

Comment:
Acceder a los elementos del array es poco clean, hay que hacer un mapeo mental. Mejor agregar variables intermedias para que sea más claro:
```suggestion
        val year = dateComponents[0]
        val month = dateComponents[1]
        val day =  dateComponents[2]
        return "$day/$month/$year"
```
Un detalle, podemos confiar en el factory y sabemos que siempre vamos a tener 3 elementos. En otro caso, faltaría el checkeo de cantidad de elementos para evitar una excepción en ejecución.